frontend/src/components/AdminCreateIssue.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { FormControl, FormLabel, Select, Input, Button, ChakraProvider, Textarea } from "@chakra-ui/react";
+
+
+    export const AdminCreateIssue = () => {
+        const url = "http://localhost:8888/api/create_issue";
+        const [adminCreateSubjects, setAdminCreateSubjects] = useState([])
+        const [adminCreateReferenceBooks, setAdminCreateReferenceBooks] = useState([])
+        const [adminCreatUnits, setAdminCreateUnits] = useState([])
```

Comment:
```suggestion
        const [adminCreateSubjects, setAdminCreateSubjects] = useState([])
        const [selectedSubject, setSelectedSubject] = useState("")
        const [adminCreateReferenceBooks, setAdminCreateReferenceBooks] = useState([])
        const [selectedReferenceBook, setSelectedReferenceBook] = useState("")
        const [adminCreatUnits, setAdminCreateUnits] = useState([])
        const [selectedUnit, setSelectedUnit] = useState("")
```

元のものを変更してしまうとmapで回せなくなるのでエラーになっていると思います。

61行目の
`
        {adminCreateSubjects.map((s) => {
`